src/api/app/controllers/webui/users/announcements_controller.rb

```diff
@@ -9,6 +9,5 @@ def create
     else
       flash.now[:error] = "Couldn't find Announcement"
     end
-    switch_to_webui2
```

Comment:
This should be kept until we get rid of it completely (which we will do later).

Reply:
why if it's not necessary? It's just necessary if it needs the webui2 views in the view path...